flow-server/src/main/java/com/vaadin/flow/internal/nodefeature/InertData.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2000-2020 Vaadin Ltd.
```

Comment:
```suggestion
 * Copyright 2000-2021 Vaadin Ltd.
```
I'm fine with 2022 as well 😝 But let's keep this consistent in so we don't miss it in the 2021 -> 2022 update